### PR TITLE
adds documentation for the usage of OpenStack Swift as Cloud Controller blobstore

### PR DESCRIPTION
**Pull request by julweber**
We added an advanced topic on the CC Swift blobstore configuration in the running cf - openstack section.

### CODE REVIEW

**Comment by julweber** [comment]
Hey guys,

could you please review this PR?

Cheers,
Julian

File: source/docs/running/deploying-cf/openstack/using_swift_blobstore.html.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<br />
+
+## Introduction ##
+
+The Cloud Controller stores user uploaded applications, buildpacks, droplets and application resources in a blobstore. There are different blobstore providers like the local file system (Local), AWS S3 (AWS) and OpenStack Swift (OpenStack). This article describes how to configure the Cloud Controller blobstore to use the Swift storage.
+
+The files uploaded by users are stored in private buckets and secured against unauthorized access. 
+If DEAs need to access these files from the Cloud Controller's blobstore, the Cloud Controller generates temporary urls to the required files and provides them to the DEA. The DEA can download the according files from the received url to execute staging tasks and delivers back the results. The generated temporary urls are only valid for a limited amount of time. This ensures that the data is secure.
+
+<strong>!!! Annotation: This feature is not available yet, because it depends on some modifications to the underlying fog gem to provide temporary urls. </strong>
```
Comment (frodenas) [inline source/docs/running/deploying-cf/openstack/using_swift_blobstore.html.md]: @julweber Is this still an issue?

**Comment by julweber** [comment]
@frodenas the fog gem contains all needed functions since version 1.16.0 . but the unmerged pull request https://github.com/cloudfoundry/cloud_controller_ng/pull/117 stops the feature from working right now since the current CC version doesn't use a new fog version. I will replace the annotation with an according text.

**Comment by frodenas** [comment]
OK, thanks @julweber

**Comment by julweber** [comment]
Thanks for the merge :)

**Pull request merged**